heliopy/data/ace.py

```diff
@@ -83,8 +86,9 @@ def mfi_h0(starttime, endtime):
     product = 'mfi_h0'
     fname = 'h0_mfi'
     version = '06'
+    units = OrderedDict([('Magnitude', u.nT), ('dBrms', u.nT), ('Q_FLAG', u.m/u.m)])
```

Comment:
u.m/u.m because I don't know right now how to enter a dimensionless unit. I will test it out a bit more!